### ISSUE TITLE
Describing size of letters in equation and scaling factor issue

### ISSUE THREAD

**Issue by terracoda** [phetsims/resistance-in-a-wire#123]
I have explored a few cases for RIAW, in order to get a sense of what the descriptions should be to describe the size of letters in the equation. There are some scaling factors that I want to make sure align with what I am seeing visually and the descriptions that I am designing.

**Description Design Question**
What is the physical size of letter R when the Resistivity, Length and Area the sliders are all set exactly at mid-range.

**Given:**
0.67 = ( 0.5 * 10 ) / 7.5

Should R be described as 
1. Size of letter R is {{comparable to}} the size of letter rho, letter L, and letter A.

or

2. Size of letter R is {{slightly larger than}} the size of letter rho, letter L, and letter A.  

As far as I can tell, visually, on page load, the letters are all the same size, so my first guess was option 1. When I set the sliders all to minimum, however, the size of R is slightly larger than the other letters. I realize these things are not linear, but I just want to make sure the descriptions are correctly matching the sim's visual representation and learning goals. I have only explored a few cases.

To help get at the root of the problem, @jessegreenberg supplied me with plots for teh R's mathematical representation and R's scaled representaion. These are available in the [RIAW A11y Design Document](https://docs.google.com/document/d/13WhBOLreoLunifv3Lcv1YVS1ryCgB7Q90iSGbnlrl3w/edit#heading=h.q471ksl4f15g).

**Root of the Problem:**
1. Do we always have the descriptions for this sim map to the quantitative values (which would result in option 2 above), or 
2. Are there special cases where we might want to fudge this a bit - e.g., option 1 above.

**Note: The scaling factor seems to be: letter size in the equation is variable=variable +1.**

**Comment by terracoda** [comment]
@ariel-phet, and @kathy-phet could you comment please.

**Comment by ariel-phet** [comment]
@terracoda I think the actual size of the letter R is not so important. Of your two choices I might go with option 1.

However, I think you could just refer to "R" independently. How R changes, its relation to the other variables is what is key.

So I think you could just say R is "extremely small, small, medium size, big, enormous" or such.

**Comment by terracoda** [comment]
@ariel-phet, this description is the dynamic description of the equation. It describes the state of the equation.

Students get alerts about R growing or shrink when the play with  the sliders.

Are the letters all supposed to me the same size  if the sliders are at the same level, or can R be different, as it is when all sliders are set to minimum?

**Comment by terracoda** [comment]
@ariel-phet, actually, I will just check with Jesse about the actual numerical size of letter R at Page Load, at MAX, and at MIN. It might be that a size difference is there at Page Load and at Max, but that it is hard to see the difference visually. It is very clear that there is a size difference when all the sliders are set to MIN. 

Regarding the actual size of letter R, that might be useful in an alert, but not so useful in the description.

**Comment by terracoda** [comment]
@ariel-phet, for example, the alerts sound like this:

**Comment by terracoda** [comment]
@ariel-phet, @kathy-phet, I discussed some edge-case explorations with @emily-phet today, and we think that the current descriptions and alerts accurately describe what is happening in the sim.

The edge cases we discussed are outlined in the design document if you would like to see them.

I am closing this issue for now. We can discuss further changes or tweaks if needed once teh current description design is implemented.
